lib/sign_in/utils.dart

```diff
@@ -33,7 +33,6 @@ Future<bool> signOut(BuildContext context) async {
   if (confirmed) {
     // Actually log out.
 
-    await services.get<NetworkService>().delete('authentication');
```

Comment:
Why was this deleted?

Reply:
This should have been `services.api.delete('authentication')` in the first place, but this results in a `401 Not authenticated` error. Trying the same request in Python or Postman, it works fine, I don't know what is different about Dart. However, it appears that even when it is successful, it does not change anything. Maybe someone with more API experience could shed some light on this?

Reply:
After moving this to the `SignOutScreen` and calling this before clearing storage and cookies, it works for me.